### PR TITLE
docs: Drop single user support

### PR DESCRIPTION
**Pull request by skabashnyuk**
## What does this pull request change?
Drop single user support
## What issues does this pull request fix or reference?
https://github.com/eclipse/che/issues/20295
https://github.com/eclipse/che/issues/20314
https://github.com/eclipse-che/che-server/pull/89
## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [x] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by themr0c** [comment]
What is the target version?

**Comment by skabashnyuk** [comment]
2.12. if that is problematic it could be postponed to 2.13

File: modules/administration-guide/partials/con_authentication-modes.adoc [hunk 1]
```diff
@@ -11,9 +11,13 @@ Single-user mode requires no authentication and anyone can access all cluster re
 
 * H2 database is used.
 
-{prod-short} deployed in single-user mode has no authentication and anyone who can access the URL of the {prod-short} deployment sees all workspaces and owns all resources. Because the deployment in this mode requires fewer containers, RAM and CPU requirements are lower. This mode is useful when the whole {prod-short} deployment is used by a single person or for lowering resources used. As the server does not authenticate, actions of multiple users logged in the same workspace can interfere with each other.
+{prod-short} deployed in single-user mode has no authentication and anyone who can access the URL of the {prod-short} deployment sees all workspaces and owns all resources. The fact that this mode has no authentication leads to it deprecation and removal. It's highly recommended migrating to multiuser mode.
```
Comment (themr0c) [inline modules/administration-guide/partials/con_authentication-modes.adoc]:
```suggestion
```

We don't need to document the previous behaviour in the docs. Therefor we can safedly drop the complete paragraph.

File: modules/administration-guide/partials/con_che-workspace-controller.adoc [hunk 2]
```diff
@@ -2,11 +2,10 @@
 [id="{prod-id-short}-workspace-controller_{context}"]
 = {prod-short} workspace controller
 
-The workspaces controller manages the container-based development environments: {prod-short} workspaces. Following deployment scenarios are available:
+The workspaces controller manages the container-based development environments: {prod-short} workspaces. Following deployment scenario is available:
```
Comment (themr0c) [inline modules/administration-guide/partials/con_che-workspace-controller.adoc]:
```suggestion
The workspaces controller manages the container-based development environments: {prod-short} workspaces. To secure the development environements with authentication, the deployment is always multiuser and multitenant.
```

Simplify, as we have only one scenario, the list becomes unnecessary.

File: modules/administration-guide/partials/con_che-workspace-controller.adoc [hunk 3]
```diff
@@ -2,11 +2,10 @@
...
 
-* *Single-user*: The deployment contains no authentication service. Development environments are not secured. This configuration requires fewer resources. It is more adapted for local installations.
 
-* *Multiuser*: This is a multitenant configuration. Development environments are secured, and this configuration requires more resources. Appropriate for cloud installations.
+* *Multiuser*: This is a multitenant configuration. Development environments are secured. Appropriate for cloud installations.
```
Comment (themr0c) [inline modules/administration-guide/partials/con_che-workspace-controller.adoc]:
```suggestion
```

Remove, concatenate with previous paragraph.

File: modules/installation-guide/partials/assembly_installing-che.adoc [hunk 4]
```diff
@@ -10,9 +10,8 @@
 This section contains instructions to install {prod}. The installation method depends on the target platform and the environment restrictions.
 
 ifeval::["{project-context}" == "che"]
-{prod-short} is available in two modes:
+{prod-short} is available in such mode:
```
Comment (themr0c) [inline modules/installation-guide/partials/assembly_installing-che.adoc]:
```suggestion
```

File: modules/installation-guide/partials/assembly_installing-che.adoc [hunk 5]
```diff
@@ -10,9 +10,8 @@
...
 
-* *Single-user*: Non-authenticated {prod-short}, lighter and suited for personal environments
 * *Multiuser*: Authenticated {prod-short} suited for organizations and developer teams
```
Comment (themr0c) [inline modules/installation-guide/partials/assembly_installing-che.adoc]:
```suggestion
```

Remove. As there is no more choice to perform.

File: modules/installation-guide/partials/con_security-considerations.adoc [hunk 6]
```diff
@@ -7,7 +7,7 @@
 
 This section explains the security impact of using different {prod-short} workspace exposure strategies.
 
-All the security-related considerations in this section are only applicable to {prod-short} in multiuser mode. The single user mode does not impose any security restrictions.
+All the security-related considerations in this section are only applicable to {prod-short} in multiuser mode.
```
Comment (themr0c) [inline modules/installation-guide/partials/con_security-considerations.adoc]:
```suggestion
```

Remove the restriction, as now it always applies.

**Comment by themr0c** [comment]
I left some suggestions @skabashnyuk. We don't need to document previous behavior, and we can remove the lists  when there is no more choice to make.

**Review by themr0c — APPROVED** [review]
Removed more occurrences of single-user, and multiuser as an option.

Tested successfully on https://workspaces.openshift.com/#https://github.com/skabashnyuk/che-docs/tree/singleuser

Not tested the procedures where I removed --multiuser, but I believe this option is obsolete for a long time.

File: modules/administration-guide/partials/con_che-workspace-controller.adoc [hunk 7]
```diff
@@ -2,13 +2,11 @@
 [id="{prod-id-short}-workspace-controller_{context}"]
 = {prod-short} workspace controller
 
-The workspaces controller manages the container-based development environments: {prod-short} workspaces. Following deployment scenarios are available:
+The workspaces controller manages the container-based development environments: {prod-short} workspaces. To secure the development environements with authentication, the deployment is always multiuser and multitenant.
```
Comment (themr0c) [inline modules/administration-guide/partials/con_che-workspace-controller.adoc]:
```suggestion
The workspaces controller manages the container-based development environments: {prod-short} workspaces. To secure the development environments with authentication, the deployment is always multiuser and multitenant.
```

File: modules/administration-guide/partials/con_che-keycloak.adoc [hunk 8]
```diff
@@ -6,7 +6,7 @@
 [id="{prod-id-short}-keycloak_{context}"]
 = {prod-short} and {identity-provider}
 
-{identity-provider} is a prerequisite to configure {prod-short} in multiuser mode. The {prod-short} administrator can choose to connect {prod-short} to an existing {identity-provider} instance or let the {prod-short} deployment start a new dedicated {identity-provider} instance.
+{identity-provider} is a prerequisite  to configure {prod-short}. The {prod-short} administrator can choose to connect {prod-short} to an existing {identity-provider} instance or let the {prod-short} deployment start a new dedicated {identity-provider} instance.
```
Comment (nickboldt) [inline modules/administration-guide/partials/con_che-keycloak.adoc]: extra space on this line

**Review by nickboldt — APPROVED** [review]
seems OK to me.

File: modules/administration-guide/partials/con_che-keycloak.adoc (hunk 8 above)
Comment (themr0c) [reply modules/administration-guide/partials/con_che-keycloak.adoc]:
```suggestion
{identity-provider} is a prerequisite to configure {prod-short}. The {prod-short} administrator can choose to connect {prod-short} to an existing {identity-provider} instance or let the {prod-short} deployment start a new dedicated {identity-provider} instance.
```

**Pull request merged**